README.md

```diff
@@ -2,4 +2,160 @@
+```
+kombucha \
+  ../my-tests/kombucha.json \
+  --snapshots-directory ../my-tests/__Snapshots__ \
+  --work-directory ../my-tests/__Work__
+```
+
+### Running via Docker
+
+⚠️ TODO!
+
+## List of checks
+
+### The `structure` check
+
+We call Kombucha’s main snapshotting behavior “the structure check”. With both the snapshot and the work (live response) JSON in memory, Kombucha will flag any JSON keys that appear in the snapshot structure, but not in the work structure. In addition, it will flag any JSON keys where the type of the value in the work structure differs from the type in the snapshot structure (eg. if the key `"foo"` used to have the value `true`, but now it has the value `99.0` or `["hello"]`).
+
+Kombucha performs this check recursively into nested JSON objects as deep as they go, and recursively into the *first element only* of nested JSON arrays, as deep as they go. The lack of coverage for “all” of an array is an issue, but we’re not sure how to solve or improve this yet.
+
+### Other checks
+
+* `array-consistency`
+   * on the topic of arrays, Kombucha can flag heterogeneous arrays in the work response, since these can be a problem for clients if they are unexpected. We call this the “array consistency” check: Kombucha will check every array in the live response to ensure that the same keys and types are present in items at the indices `[1...]` as are present at index `0`.
+* `empty-arrays`
+   * much more simply, Kombucha can also flag empty arrays (`"key": []`) in the response, as this may also cause problems with clients, or may represent an opportunity to optimize the API.
+* `empty-objects`
+   * likewise, flag empty object literals (`"key": {}`) in the response
+* `flag-new-keys`
+   * the inverse of the `structure` check: flag JSON keys that appear in the “work” (live response) JSON, but **not** in the snapshot. This may indicate that your snapshot is out of date, or can be used to generate API diffs for endpoints that are under active development.
+* `string-bools`
+   * flag the values `"true"` and `"false"` (which should probably be `true` and `false` instead)
+* `string-numbers`
+   * flag, for example, `"99.0"` (as opposed to `99.0`). This check should be used sparingly as there are plenty of pseudo-numeric values that nevertheless are best represented as strings (for example: opaque identifiers, or zip codes in the United States)
+
+### Default checks
+
+If no [`__preferences` are specified](#customizing-output), Kombucha runs with a default series of checks that should be useful for most situations:
+
+* **errors**: `structure`
+* **infos**: `empty-arrays`, `empty-objects`, `flag-new-keys`, `string-bools`
+* **warnings**: `array-consistency`
+
+## Implementation details / Development / Contributing
+
+⚠️ TODO!
```

Comment:
Heavy details can wait, but I would like to acknowledge the basic functional “stuff” here as well as credit pointfreeco for the snapshot testing inspiration.